src/read.c

```diff
@@ -787,7 +787,7 @@ static avifResult avifCheckItemID(const char * boxFourcc, uint32_t itemID, avifD
     return AVIF_RESULT_OK;
 }
 
-static avifResult avifMetaFindOrCreateItem(avifMeta * meta, uint32_t itemID, avifDecoderItem ** item)
+static avifResult avifMetaFindOrCreateItem(avifMeta * meta, uint32_t itemID, avifDiagnostics * diag, avifDecoderItem ** item)
```

Comment:
Nit: By convention `avifDiagnostics * diag` is usually the last parameter.

Reply:
Done